search/shortest.go

```diff
@@ -0,0 +1,263 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package search
+
+import (
+	"math"
+	"math/rand"
+
+	"github.com/gonum/graph"
+	"github.com/gonum/matrix/mat64"
+)
+
+// Shortest is a shortest-path tree created by the DijkstraFrom function.
+type Shortest struct {
+	// from holds the source node given to
```

Comment:
Sorry, the expand on this, shorter line length arguably eases comprehension (the width I tend to format to is close to a newspaper column width) and in these cases keeps the text near the field name and type. In godoc comments, this is less of an issue because we are providing text to be formatted by godoc. Further, in the case of exported structs and interfaces, the field comments are not reformatted by godoc, so we need to provide a reasonable default for many terminals and browsers.